errors.go

```diff
@@ -109,3 +109,10 @@ type ErrUpsertMissingTarget struct{}
 func (e *ErrUpsertMissingTarget) Error() string {
 	return "has a missing conflict target"
 }
+
+// ErrRowIDNotAllowed indicate a reference to the columns rowid, _rowid_, or oid in an INSERT, UPDATE or CREATE statement.
```

Comment:
New error to indicate that you cannot use the `rowid`, `oid`and `_rowid_` identifiers